docker-image/Dockerfile

```
@@ -41,12 +42,24 @@ RUN apt-get update \
         docker-ce-cli \
         python3.7 \
         python3-pip \
+        rubygems \
 && apt-get clean \
 \
 && python3.7 -m pip install --upgrade \
         pip setuptools wheel \
```

```suggestion
        lxml pip setuptools wheel \
```
It turned out that lxml is required (I've built image locally with `--no-cache=true`).